### PR TITLE
Add drag and drop

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

File: features/file-system-access.yml [hunk 1]
```diff
@@ -4,8 +4,6 @@ spec: https://wicg.github.io/file-system-access/
 caniuse: native-filesystem-api
 group: file-system
 compat_features:
-  # api.DataTransferItem.getAsFileSystemHandle is likely to be part of a drag and drop feature, but it can't be in both features at once. See https://github.com/web-platform-dx/web-features/issues/1173 for details
```
Comment (jamesnw) [inline features/file-system-access.yml:7]: I agree it makes more sense in Drag and drop, but should we move this comment there?

File: features/draganddrop.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name: Drag and Drop
+description: "The Drag and Drop API lets web applications add dragging and dropping of elements and external files such as images onto web pages. Developers can customize which elements can become draggable, the type of feedback the draggable elements produce, and the droppable elements."
```
Comment (jamesnw) [inline features/draganddrop.yml:2]:
```suggestion
description: "The Drag and Drop API lets users drag and drop elements and external files such as images onto web pages. Developers can customize which elements can become draggable, the type of feedback the draggable elements produce, and the droppable elements."
```

I don't think we can or need to try to escape the circular definition here. I noted MDN uses `drag-and-drop`, but that doesn't seem to be the preferred term per Google Trends. https://trends.google.com/trends/explore?date=today%205-y&q=drag-and-drop,drag%20and%20drop&hl=en

**Review by jamesnw — APPROVED** [review]
The feature set looks good to me!

File: features/file-system-access.yml (hunk 1 above)
Comment (autonome) [reply features/file-system-access.yml:7]: Good call, I moved it over and reworded.

**Pull request merged**